Fix some typos in a configuration variable name.

Kerberos RT #3468
Debian bug #347643

=== krb5/src/config-files/krb5.conf.M
==================================================================
--- krb5/src/config-files/krb5.conf.M	(revision 2079)
+++ krb5/src/config-files/krb5.conf.M	(local)
@@ -196,8 +196,8 @@
 
 .IP udp_preference_limit
 When sending a message to the KDC, the library will try using TCP
-before UDP if the size of the message is above "udp_preference_list".
-If the message is smaller than "udp_preference_list", then UDP will be
+before UDP if the size of the message is above "udp_preference_limit".
+If the message is smaller than "udp_preference_limit", then UDP will be
 tried before TCP.  Regardless of the size, both protocols will be
 tried if the first attempt fails.
 
